src/vs/platform/windows/electron-main/windowsMainService.ts

```diff
@@ -1060,17 +1056,18 @@ export class WindowsMainService extends Disposable implements IWindowsMainServic
 			}
 		}
 
-		let authority = '';
+		let remoteAuthority = openConfig.remoteAuthority;
 		for (const extensionDevelopmentPath of extensionDevelopmentPaths) {
 			if (extensionDevelopmentPath.match(/^[a-zA-Z][a-zA-Z0-9\+\-\.]+:/)) {
 				const url = URI.parse(extensionDevelopmentPath);
-				if (url.scheme === Schemas.vscodeRemote) {
-					if (authority) {
-						if (url.authority !== authority) {
+				const devPathRemoteAuthority = getRemoteAuthority(url);
```

Comment:
I would explicitly call this `extensionDevelopmentPathRemoteAuthority`, I couldn't understand this at first.